From: Julian Ospald <hasufell@gentoo.org>
Date: Wed Dec 31 16:16:51 UTC 2014
Subject: use pkg-config

	This also fixes build with libressl.

--- libeXosip2-4.0.0/configure.ac
+++ libeXosip2-4.0.0/configure.ac
@@ -376,9 +375,10 @@
 	enable_openssl=$enableval,enable_openssl="yes")
 
 if test "$enable_openssl" = "yes" ; then
-	AC_CHECK_HEADERS(openssl/ssl.h)
-	AC_CHECK_LIB(ssl,SSL_CTX_new,[EXOSIP_LIB="$EXOSIP_LIB -lssl"])
-	AC_CHECK_LIB(crypto,X509_get_issuer_name,[EXOSIP_LIB="$EXOSIP_LIB -lcrypto"])
+	PKG_CHECK_MODULES(SSL, libssl libcrypto,
+					  [EXOSIP_LIB="$EXOSIP_LIB $SSL_LIBS"
+					   EXOSIP_FLAGS="$EXOSIP_FLAGS $SSL_CFLAGS"],
+					   AC_MSG_ERROR(["Couldn't find libssl/libcrypto"]))
 fi
 
 AC_ARG_ENABLE(srvrec,
