From: Julian Ospald <hasufell@gentoo.org>
Date: Sat Jan  3 02:47:59 UTC 2015
Subject: add build system switches

--- zuluCrypt-4.7.4/CMakeLists.txt
+++ zuluCrypt-4.7.4/CMakeLists.txt
@@ -18,6 +18,9 @@
 
 INCLUDE(FindPkgConfig)
 
+option(WITH_PWQUALITY "Build with pwquality support" 1)
+option(WITH_TCPLAY "Build with tcplay support" 1)
+
 if( NOT NOGNOME )
 	pkg_check_modules( LIBSECRET libsecret-1 )
 else( NOT NOGNOME )
@@ -30,8 +33,12 @@
 
 pkg_check_modules( CRYPTSETUP libcryptsetup )
 pkg_check_modules( BLKID blkid )
-pkg_check_modules( PWQUALITY pwquality )
-pkg_check_modules( TCPLAY tcplay )
+if ( WITH_PWQUALITY )
+	pkg_check_modules( PWQUALITY pwquality )
+endif ( WITH_PWQUALITY )
+if ( WITH_TCPLAY )
+	pkg_check_modules( TCPLAY tcplay )
+endif ( WITH_TCPLAY )
 
 set( PGR_VERSION "4.7.4" )
 set( LIB_VERSION "1.2.0" )
