From: Julian Ospald <hasufell@gentoo.org>
Date: Sat Jan  3 02:47:59 UTC 2015
Subject: fix qt build

--- zuluCrypt-4.7.4/CMakeLists.txt
+++ zuluCrypt-4.7.4/CMakeLists.txt
@@ -150,7 +150,7 @@
 
 		add_definitions( ${Qt5Widgets_DEFINITIONS} )
 	else()
-		find_package( Qt4 REQUIRED )
+		find_package( Qt4 REQUIRED QtCore QtGui QtNetwork )
 
 		add_definitions( -I${Qt4_INCLUDE_DIR} )
 		add_definitions( -I${QT_HEADERS_DIR} )
--- zuluCrypt-4.7.4/zuluCrypt-gui/CMakeLists.txt
+++ zuluCrypt-4.7.4/zuluCrypt-gui/CMakeLists.txt
@@ -92,12 +92,12 @@
 
 if( QT5 )
 	if( library_pwquality )
-		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${blkid} ${library_pwquality} )
+		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${Qt5Network_LIBRARIES} ${blkid} ${library_pwquality} )
 	else( library_pwquality )
-		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${blkid} ${kwallet_library} )
+		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${Qt5Network_LIBRARIES} ${blkid} ${kwallet_library} )
 	endif( library_pwquality )
 
-	target_link_libraries( zuluCrypt-gui sharedObject zuluCryptPluginManager -lQt5Network zuluCryptMD5 )
+	target_link_libraries( zuluCrypt-gui sharedObject zuluCryptPluginManager zuluCryptMD5 )
 else()
 	if( library_pwquality )
 		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${QT_LIBRARIES} ${blkid} ${library_pwquality} )
@@ -105,7 +105,7 @@
 		TARGET_LINK_LIBRARIES( zuluCrypt-gui lxqtwallet ${QT_LIBRARIES} ${blkid} ${kwallet_library} )
 	endif( library_pwquality )
 
-	target_link_libraries( zuluCrypt-gui sharedObject zuluCryptPluginManager -lQtNetwork zuluCryptMD5 )
+	target_link_libraries( zuluCrypt-gui sharedObject zuluCryptPluginManager zuluCryptMD5 )
 
 endif()
 
--- zuluCrypt-4.7.4/zuluMount-gui/CMakeLists.txt
+++ zuluCrypt-4.7.4/zuluMount-gui/CMakeLists.txt
@@ -52,7 +52,7 @@
 
 	add_executable( zuluMount-gui ${MOC} ${UI} ${SRC} ${TRAY_RC_SRCS} )
 
-	TARGET_LINK_LIBRARIES( zuluMount-gui sharedObject lxqtwallet  ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${Qt5Widgets_LIBRARIES} ${blkid} zuluCryptPluginManager -lQt5Network )
+	TARGET_LINK_LIBRARIES( zuluMount-gui sharedObject lxqtwallet  ${Qt5Widgets_LIBRARIES} ${Qt5Core_LIBRARIES} ${Qt5Widgets_LIBRARIES} ${Qt5Network_LIBRARIES} ${blkid} zuluCryptPluginManager )
 
 	set_target_properties( zuluMount-gui PROPERTIES INSTALL_RPATH "${LIB_PATH}" )
 else()
@@ -73,7 +73,7 @@
 
 	add_executable( zuluMount-gui ${MOC} ${UI} ${SRC} ${TRAY_RC_SRCS} )
 
-	TARGET_LINK_LIBRARIES( zuluMount-gui sharedObject lxqtwallet ${QT_LIBRARIES} ${blkid} zuluCryptPluginManager -lQtNetwork )
+	TARGET_LINK_LIBRARIES( zuluMount-gui sharedObject lxqtwallet ${QT_LIBRARIES} ${blkid} zuluCryptPluginManager )
 
 	set_target_properties( zuluMount-gui PROPERTIES INSTALL_RPATH "${LIB_PATH}" )
 endif()
